video-app/src/app/library/home/home.component.html

```diff
@@ -1,79 +1,168 @@
 <div class="jumbotron jumbotron-fluid">
   <div class="container">
     <h1 class="display-4">Video App</h1>
-    <p class="lead">You can watch your favourite videos from different platforms all in one place.</p>
+    <p class="lead">
+      You can watch your favourite videos from different platforms all in one place.
+    </p>
   </div>
 </div>
 
 <div class="navbar-container">
```

Comment:
to moglabys wyniesc do osobnego komponentu, ktory na Input() przyjmuje aktualne wartosci i na Output() emitujesz zmiany wartosci:)

Reply:
niezrobione